client/app/services/custom-button/custom-button.component.js

```diff
@@ -1,24 +1,51 @@
+/* eslint comma-dangle: 0 */
+
 import './_custom-button.sass'
-import templateUrl from './custom-button.html'
+import './_custom-button-menu.sass'
+import templateInline from './custom-button.html'
+import templateMenu from './custom-button-menu.html'
 
-export const CustomButtonComponent = {
+const base = {
   bindings: {
     customActions: '<',
     serviceId: '<',
-    vmId: '<'
+    vmId: '<',
+    displayFor: '<?',
   },
   controller: CustomButtonController,
   controllerAs: 'vm',
-  templateUrl
 }
 
+export const CustomButtonComponent = Object.assign({}, base, {
```

Comment:
This should really be just `= {...base, templateUrl: templateInline}` but alas the current build system does not support `...` yet.